CBLClient/Apps/CBLTestServer-Android/app/src/main/java/com/couchbase/CouchbaseLiteServ/AndroidContext.java

```diff
@@ -0,0 +1,59 @@
+package com.couchbase.CouchbaseLiteServ;
+
+import com.couchbase.javacommon.Context;
+import com.couchbase.javacommon.log.Log;
+
+import java.io.File;
+import java.io.InputStream;
+import java.net.Inet4Address;
+import java.net.InetAddress;
+import java.net.NetworkInterface;
+
+import java.util.Enumeration;
+
+public class AndroidContext implements Context {
+    private static final String TAG = "AndroidContext";
```

Comment:
Ohhh... I don't think you should do this.  Contexts are big hairy object and they don't belong to you.  They belong to the container.  You should not be subclassing them.

What would you think of creating a TestServerContext, and passing the application context into its constructor.  You could do that in the Application object...